### PR TITLE
Select only case type for case exports

### PR DESCRIPTION
**Pull request by esoergel**
@biyeun @orangejenny @dannyroberts
https://manage.dimagi.com/default.asp?262543
This drops the application drilldown from case export creation.  Now you only select a case type.

### CODE REVIEW

**Comment by dannyroberts** [comment]
Error looks related

```
Running "mocha:app_manager#fields" (mocha) task
Testing: http://localhost:8000/mocha/app_manager/fields
  DrilldownToFormController - Initialization
  1) Basic Form init
  2) Multiple App Types init
  3) Case Types init
  ✓ registers server error 
  ✓ registers HTTP error 
  2 passing (6ms)
  3 failing
  1) DrilldownToFormController - Initialization Basic Form init:
     AssertionError: expected [] to equal undefined
      at http://localhost:8000/static/chai/chai.js:210
      at assertEqual (http://localhost:8000/static/chai/chai.js:752)
      at http://localhost:8000/static/chai/chai.js:3854
      at http://localhost:8000/static/chai/chai.js:2507
      at http://localhost:8000/static/app_manager/spec/app_data_drilldown.DrilldownToFormController.init.spec.mo.js:49
  2) DrilldownToFormController - Initialization Multiple App Types init:
     AssertionError: expected [] to equal undefined
      at http://localhost:8000/static/chai/chai.js:210
      at assertEqual (http://localhost:8000/static/chai/chai.js:752)
      at http://localhost:8000/static/chai/chai.js:3854
      at http://localhost:8000/static/chai/chai.js:2507
      at http://localhost:8000/static/app_manager/spec/app_data_drilldown.DrilldownToFormController.init.spec.mo.js:98
  3) DrilldownToFormController - Initialization Case Types init:
     AssertionError: expected [ { id: 'case', text: 'Case' } ] to equal undefined
      at http://localhost:8000/static/chai/chai.js:210
      at assertEqual (http://localhost:8000/static/chai/chai.js:752)
      at http://localhost:8000/static/chai/chai.js:3854
      at http://localhost:8000/static/chai/chai.js:2507
      at http://localhost:8000/static/app_manager/spec/app_data_drilldown.DrilldownToFormController.init.spec.mo.js:142
>> 3/5 tests failed (0.01s)
Warning: Task "mocha:app_manager#fields" failed. Use --force to continue.
```

File: corehq/apps/export/forms.py [hunk 1]
```diff
@@ -160,24 +160,24 @@ def __init__(self, has_form_export_permissions, has_case_export_permissions, *ar
...
                     crispy.Field(
-                        'app_type',
-                        placeholder=_("Select Application Type"),
-                        ng_model="formData.app_type",
-                        ng_change="updateAppChoices()",
-                        ng_required="true",
+                        'application',
+                        placeholder=_("Select Application"),
+                        ng_model="formData.application",
+                        ng_change="formData.model_type === updateModuleChoices()",
```
Comment (orangejenny) [inline corehq/apps/export/forms.py]: Comparing this with old line 177, this looks like it should just be `updateModuleChoices()`...?

Comment (esoergel) [reply corehq/apps/export/forms.py]: you're right - will update.

**Comment by esoergel** [comment]
This should be g2m now, I think

**Review by orangejenny — APPROVED** [review]

**Pull request merged**